onnx/defs/schema.h

```diff
@@ -1098,6 +1099,27 @@ class OpSchema final {
       std::set<std::string>* updated_ops = nullptr) const;
   void UpdateFunctionProtoOpsetImportVersion(FunctionProto& function_proto, int opset_version) const;
 
+  /**
+   * @brief A common function to generate a prefix string for use in fail_check during the verify function.
+   * @param  node_name If empty, the returned string will not include the node name.
+   * @return std::string The prefix string.
+   */
+  std::string VerifyFailPrefix(std::string_view node_name) const;
```

Comment:
I would keep this private

Reply:
Hi, this is the private scope which beginning at L1092. ;D

Reply:
Sorry I missed that. Thanks!